lib/qualys/was/vulnerability.rb

```diff
@@ -45,24 +48,66 @@ def method_missing(method, *args)
         return
       end
 
-      method_name = method.to_s
+      # Any fields where a simple .camelcase() won't work we need to translate,
+      # this includes acronyms (e.g. :cwe would become 'Cwe') and simple nested
+      # tags.
+      translations_table = {
+        access_paths:       'ACCESS_PATH/URL',
+        ajax:               'AJAX',
+        authentication:     'AUTHENTICATION',
+        ignored:            'IGNORED',
+        potential:          'POTENTIAL',
+        url:                'URL',
```

Comment:
I don't think this needs to be in the translation table.

You can change this to something like:

```ruby
method_name = 
  if translation_table[method]
    translation_table[method]
  else
    method.to_s.upcase
  end
```